Fix build failure with Clangs from Xcode 7 and earlier.
This has been reported to the developer by email.
--- misc/consts.h.orig	2019-02-22 19:36:36.000000000 -0600
+++ misc/consts.h	2019-10-09 05:57:18.000000000 -0500
@@ -345,7 +345,7 @@
 #define MIN_STORED_TIME_VALUE	( YEARS_TO_SECONDS( 1995 - 1970 ) )
 #define MAX_TIME_VALUE			( YEARS_TO_SECONDS( 2036 - 1970 ) )
 
-#if ( defined( __clang__ ) && ( __clang_major__ > 5 ) ) || \
+#if ( defined( __clang__ ) && ( ( __clang_major__ > 5 && !defined(__apple_build_version__) ) || __apple_build_version__ >= 8000000 ) ) || \
 	( defined( __GNUC__ ) && ( __GNUC__ > 6 ) ) || \
 	( defined( _MSC_VER ) && VC_GE_2017( _MSC_VER ) )
   #define CURRENT_TIME_VALUE	( ( DATE_YEAR + DATE_MONTH + DATE_DAY - 30 ) * 86400 )
